src/ui/components/CouncilSummaryTeaser.tsx

```diff
@@ -26,7 +27,7 @@ export function CouncilSummaryTeaser(props: PropTypes) {
         <LoadingBox />
       ) : summary ? (
         <View style={styles.container}>
-          <Card style={[styles.item, styles.left]} disabled>
+          <View style={[styles.card, {borderColor: colors.backdrop}]}>
```

Comment:
The same, the `Card` component should take care of using the right color base on the current theme, no need to add custom styles